hbf/tests/test_getAddress.py

```diff
@@ -13,7 +13,7 @@ def test_getAddress():
         ZipCode,
         SearchBoxHalfWidth,
         RequestHeaders,
-    ) = getInput("./hbf/tests/res.inp")
+    ) = getInput("./tests/res.inp")
```

Comment:
I wish you @AmritPatel would just make up your damn mind already on this folder structure....